Makefile

```diff
@@ -185,6 +185,32 @@ build/$(PI_CROSS_ARCH)/sysroot/usr: check-env-sysroot
 pi: pi-sysroot
 	$(MAKE) CROSS_ARCH=raspberry_pi SYSROOT="$(CURDIR)/build/$(PI_CROSS_ARCH)/sysroot" all-cross
 
+ARM_LINUX_GNUEABI_CROSS_ARCH := arm-linux-gnueabi
```

Comment:
I wonder if we should find a way to move some of these out of the main makefile.

Reply:
I will try to do that in a separate PR.